### DISCUSSION TITLE
Below the battery icon in the menu.

### DISCUSSION THREAD

**Discussion by f4axk** [egzumer/uv-k5-firmware-custom#446] [General]
Below the battery icon in the menu.
Menu 3,4,5,6 it is displayed 0
Menu 54 it is displayed 1
Maybe normal?? , maybe bytes gained if deleted

![menu_0](https://github.com/egzumer/uv-k5-firmware-custom/assets/68162981/9e339790-f43d-4939-b190-1502818ff3b3)
![menu_1](https://github.com/egzumer/uv-k5-firmware-custom/assets/68162981/92af142f-e9d2-4e69-b640-e3c8bb8b2dd7)

**Comment by radonslide** [comment]
This shows you the number of the selected option in e.g. D LIST if there are several sub-options to choose from

**Reply by f4axk** [reply]
thanks for the information
73s
